db/seeds.rb

```diff
@@ -1,21 +1,68 @@
-User.destroy_all
-PathChallenge.destroy_all
 Pathway.destroy_all
 Challenge.destroy_all
-
+UserChallenge.destroy_all
+User.destroy_all
 puts "RUNNING SEED FILE"
 puts "-----------------"
-user = User.create!(first_name: "Ben", last_name: "Simpson", email: "ben@ben.com", password: "Testpassword1!", role: :mentor)
-puts "One user created"
 
-pathway = Pathway.create!(title: "Test 1", details: "This is a test pathway", difficulty: "beginner", user_id: user.id)
-puts "One pathway created"
+@mentor = User.create!(
+    first_name: "Mentor", last_name: "Test", email: "mentor@test.com", password: "Testpassword1!", role: "mentor")
+
+@mentee = User.create!(
+    first_name: "Mentee", last_name: "Test", email: "mentee@test.com", password: "Testpassword1!", role: "mentee")
+
+level = ["beginner", "intermidiate", "professional", "expert" ]
+
+mark = ["1", "2", "3", "4", "5" ]
 
-challenge = Challenge.create!(title: "Database number 1", details: "This is a question that needs answers regarding the database", user_id: User.last.id)
+def CreateChallenge(order)
+    Challenge.create!(
+        title: "Q #{order} Public Cloud and Private Cloud provider",
```

Comment:
`title: "Challenge, #{i + 1}!",`
So that challenges are named but not indexed at 0. I.e
Challenge 1, Challenge  2...
Instead of Challenge 0, Challenge 1 ...

Reply:
Also included the details above!